Data-Structures/src/main/java/Data/Structures/LinkedList/Llist.java

```diff
@@ -0,0 +1,49 @@
+package Data.Structures.LinkedList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Llist {
+    public  Node head;
+    public  Node current;
+
+    public Llist(){
+
```

Comment:
Per the instructions, when creating a new LList the head should be set to `null`

Reply:
you don't have to. because the Llist is null. you can see my test.